site/access-control.md

```diff
@@ -555,6 +555,12 @@ A special [cache backend](https://github.com/rabbitmq/rabbitmq-server/tree/v3.12
 can be used in [combination](#combined-backends) with other backends to significantly
 reduce the load they generate on external services, such as LDAP or HTTP servers.
 
+The available tier 1 alternative authorisation backends are
+* [LDAP](ldap.html)
+* [HTTP](https://github.com/rabbitmq/rabbitmq-auth-backend-http)
+* [AMQP](https://github.com/rabbitmq/rabbitmq-auth-backend-amqp)
+* [OAuth2](oauth2.html)
+
```

Comment:
Are these plugins?  if so the opening sentence should be "The available tier 1 alternative authorisation bankend plugins are: 

Also, if we are going to list the alternative authorisation backends in this section in a bulleted list, then we should be consistent and do the same for the "authentication" ones and "combined" ones so I would lay out this section like this to give it structure/flow:

**[Alternative Authentication and Authorisation Backends]**(https://www.rabbitmq.com/access-control.html#backends)

Authentication and authorisation are pluggable. Plugins can provide implementations of

authentication ("authn") backends: they determine client identity and decide whether the client should be allowed to connect.
authorisation ("authz") backends: they determine whether an identified (authenticated) client is authorized to perform a certain operation.

The available tier 1 alternative authentication backends plugins are:
 - ....
 - ....

The available tier 1 alternative authorisation backends plugins are:
* [LDAP](ldap.html)
* [HTTP](https://github.com/rabbitmq/rabbitmq-auth-backend-http)
* [AMQP](https://github.com/rabbitmq/rabbitmq-auth-backend-amqp)
* [OAuth2](oauth2.html)

The plugins that provide both the authentication and authorisation plugins are:

 - .....
 - ....

Then talk about the special case plugins...

Also, the new ones you have listed as part of this PR @MirahImage are provided [here](https://www.rabbitmq.com/access-control.html#topic-authorisation) as part of this: "If
 a different authorisation backend (e.g. [LDAP](https://www.rabbitmq.com/ldap.html), [HTTP](https://github.com/rabbitmq/rabbitmq-auth-backend-http), [AMQP](https://github.com/rabbitmq/rabbitmq-auth-backend-amqp), [OAuth2](https://www.rabbitmq.com/oauth2.html)) is used, please refer to the documentation of those backends."
 
 If we are going to list them now as part of this PR, then have the sentence above where they are listed link back to the: https://www.rabbitmq.com/access-control.html#backends section

Reply:
They are backends provided by plugins. The guide does not talk about plugins per se, it talks about backends.